.stylelintrc.json

```diff
@@ -2,6 +2,7 @@
   "extends": "stylelint-config-sass-guidelines",
   "rules": {
     "max-nesting-depth": 2,
+    "selector-max-compound-selectors": 4,
```

Comment:
This change was made to support [this selector](https://github.com/kbase/ui-refresh-test/pull/57/files#diff-d8f418fb21e7842d8e14b368296ed3d64039114df066036cb6a6f8d78f09cb88R59), which transpiles to `.table-wrapper table thead th`, all parts of which are required to target the correct element.

Reply:
It's good to have a limit, and this limit seems just as good to me as what it was before.